mods/cnc/chrome/ingame-infostats.yaml

```diff
@@ -153,3 +168,13 @@ Container@SKIRMISH_STATS:
 									ImageName: kick
 									X: 6
 									Y: 8
+						Checkbox@MUTE:
+							X: 418
+							Width: 20
+							Height: 25
+							Shadow: True
+							Checkmark: mute
+							Background:
+							TooltipContainer: TOOLTIP_CONTAINER
+							TooltipText: Mute this player
```

Comment:
This line (and all the others like it) is not needed anymore because the tooltip text is set in the chrome logic.